### PR TITLE
Merge branch 'google' into 'master'

### PR DESCRIPTION
**Pull request by hoisie**

### CODE REVIEW

File: shadows/framework/src/main/java/org/robolectric/shadows/ShadowRangingResult.java [hunk 1]
```diff
@@ -2,210 +2,23 @@
...
-import android.net.wifi.rtt.ResponderLocation;
-import android.os.Build;
 import android.os.Build.VERSION_CODES;
-import org.robolectric.RuntimeEnvironment;
 import org.robolectric.annotation.Implements;
-import org.robolectric.util.ReflectionHelpers;
 
 /** Shadow for {@link android.net.wifi.rtt.RangingResult}. */
 @Implements(value = RangingResult.class, minSdk = VERSION_CODES.P)
+@Deprecated
```
Comment (MGaetan89) [inline shadows/framework/src/main/java/org/robolectric/shadows/ShadowRangingResult.java:10]: Shouldn't there be a comment to indicate what the alternative is (if any)?

**Comment by hoisie** [comment]
Yeah there are a couple issues with the commits, I will fix them and then retry the merge. I can update that deprecated error message.

**Comment by MGaetan89** [comment]
@hoisie will `master` also be merged into `google`? For example, to pick up the changes for https://github.com/robolectric/robolectric/blob/google/.github/workflows/check_code_style.yml as we discussed in an other PR

**Comment by hoisie** [comment]
@MGaetan89 yup, working on that now, could take some time to verify that all the recent changes to `master` don't cause issues in Google's tests

**Pull request merged**